Misc/NEWS.d/next/Core and Builtins/2019-09-12-16-36-16.bpo-38106.4pApn7.rst

```diff
@@ -0,0 +1,6 @@
+Fix race in PyThread_release_lock that was leading to memory corruption and
+deadlocks.
```

Comment:
I'm not sure how it is rendered. I suggest to merge the sentences into a single paragraph.

Reply:
Ok, I've amended the patch with the following interdiff:

```diff
--- a/Misc/NEWS.d/next/Core and Builtins/2019-09-12-16-36-16.bpo-38106.4pApn7.rst       
+++ b/Misc/NEWS.d/next/Core and Builtins/2019-09-12-16-36-16.bpo-38106.4pApn7.rst       
@@ -1,6 +1,5 @@
 Fix race in PyThread_release_lock that was leading to memory corruption and
-deadlocks.
-
-The fix applies to POSIX systems where Python locks are implemented with mutex
-and condition variable because POSIX semaphores are either not provided, or are
-known to be broken. One particular example of such system is macOS.
+deadlocks. The fix applies to POSIX systems where Python locks are implemented
+with mutex and condition variable because POSIX semaphores are either not
+provided, or are known to be broken. One particular example of such system is
+macOS.
```

For the reference sometimes changelog entries are added as several paragraphs and it renders e.g. like [this](https://docs.python.org/3.8/whatsnew/changelog.html#changelog) (search for "bpo-33185: Fixed regression when running pydoc with the -m switch" there).